### DISCUSSION TITLE
Exfat microSd problem

### DISCUSSION THREAD

**Discussion by Res777** [rg35xx-cfw/Koriki#125] [Q&A]
I accidentally inserted an exfat microSD into slot 2, the system remained in bootloop, I removed it... but when I put it back on Windows it is accessible but impossible to format because it is write protected, nothing changes either with regedit not even with diskpart...What can I do?

**Comment by Lalarian** [comment]
1) Try these methods
https://www.easeus.com/partition-master/how-to-format-a-write-protected-sd-card.html#repair_device


2) If none of these helped, try booting into a live linux system and unprotecting / formatting the SD from there
a)
sudo su
ls -al /dev/sd* #check which numbers for * it returns using:
fdisk -l /dev/sd* (go through all the numbers, until you find the right drive)
then use:
sudo hdparm -r0 /dev/sdN  (with X being the number of your SD you found before

b) mkfs.fat32 /dev/sdN using same number as above

3) If neither 1) or 2) worked: 
"Write Protected" sometimes results from a fail of the microcontroller inside the SD (same für USB thumb drives..).
I sometimes could fix this flashing the firmware of a device new (with reset to defaults enabled) using the contoller'sS vendor tools (like Phison) with having the "reset to defualt" option enabled. But it isn't really worth, because crappy media like this will fail again, and again .. So if neither 1) or 2) helped, best is to throw it into the garbage.

**Reply by Res777** [reply]
Thanks, I'll try the linux procedure you suggested. However the microsd is not stock, it is a 200gb patriot that had no problem... but with chipgenius I can not detect the controller.

**Reply by Res777** [reply]
Update:
No way, nothing, no terminal command solves the problem (I've tried others too...like rooted android and twrp) and it always returns a write protection error, like windows.
Damn me when I inserted this exfat microsd into the rg35xx!!!! don't try this at home!
